python/pyspark/sql/connect/dataframe.py

```diff
@@ -51,24 +52,11 @@
 
 
 class GroupingFrame(object):
```

Comment:
not related to this PR, but shall we rename it `GroupedData` to be the same with pyspark?

Reply:
Renamed